.deploy/nais/app-preprod.yaml

```diff
@@ -1,12 +1,14 @@
 apiVersion: "nais.io/v1alpha1"
 kind: "Application"
 metadata:
-  name: familie-ba-mottak
+  name: familie-baks-mottak
```

Comment:
Man vil vel få azure-endringer, så må få åpnet opp for familie-baks-mottak i:
- familie-ba-infotrygd
- pdl
- familie-integrasjoner
- familie-ba-sak

Reply:
Og familie-dokument